src/workloads/replication/startup/3_0_Reads.yml

```diff
@@ -0,0 +1,47 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: >
+ Issues dummy reads against both databases used in the light and the heavy phases.
```

Comment:
Why do we need these reads?

Reply:
Because DSI restarts mongod between tests, so we need DSI to issue another restart after the last step so I have added a dummy workloads at the end.